notifications.bs

```diff
@@ -108,8 +108,8 @@ not enough space to display the <a>notification</a> itself. It <em>may</em> also
 the <a>notification</a>, but then it should have less visual priority than the
 <a for=notification>image resource</a> and <a for=notification>icon resource</a>.
 
-<p>A <a>notification</a> <em>can</em> have a
-<dfn for=notification id=vibration-pattern>vibration pattern</dfn>.
+<p>A <a>notification</a> has an associated
+<dfn for=notification id=vibration-pattern>vibration pattern</dfn>, which is initially « ».
```

Comment:
There's a sentence in "create a notification" which says "(Otherwise vibration pattern is not set.)" which seems not so accurate now.